frontend/src/services/comment-api.service.ts

```diff
@@ -27,6 +27,32 @@ class CommentApi {
       method: HttpMethod.DELETE,
     });
   }
+
+  public uploadAudioComment(
+    file: File,
+    pageId: string,
+  ): Promise<{ url: string }> {
+    const fd = new FormData();
+    fd.append('audio', file, file.name);
+
+    return this._httpService.load(
+      `${this.BASE}/${pageId}/upload-audio-comments`,
+      {
+        method: HttpMethod.POST,
+        payload: fd,
+      },
+    );
+  }
+
+  public transcriptAudioComment(file: File): Promise<{ comment: string }> {
+    const fd = new FormData();
+    fd.append('audio', file, file.name);
+
+    return this._httpService.load(`${this.BASE}/transcript-comments`, {
+      method: HttpMethod.POST,
+      payload: fd,
+    });
+  }
```

Comment:
Class already is called **Comment**, you don't need to repeat it's name in the methods
```suggestion
  public uploadAudio(
    file: File,
    pageId: string,
  ): Promise<{ url: string }> {
    const fd = new FormData();
    fd.append('audio', file, file.name);

    return this._httpService.load(
      `${this.BASE}/${pageId}/upload-audio-comments`,
      {
        method: HttpMethod.POST,
        payload: fd,
      },
    );
  }

  public transcriptAudio(file: File): Promise<{ comment: string }> {
    const fd = new FormData();
    fd.append('audio', file, file.name);

    return this._httpService.load(`${this.BASE}/transcript-comments`, {
      method: HttpMethod.POST,
      payload: fd,
    });
  }
```